Week-10/shortly/src/utils/utils.js

```diff
@@ -0,0 +1,26 @@
+export const isUrlValid = (url) => {
+  var res = url.match(
+    /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/g
+  );
+  return res != null;
+};
+
+export const getShortenResponse = async (url) => {
```

Comment:
Kindly incorporate custom hook for API calls. 

https://codesandbox.io/s/eloquent-panna-qt3eee?file=/src/useApi.js